static_files/grafana-config/dashboards/prysm-latency-dashboard.json

```diff
@@ -0,0 +1,2028 @@
+{
```

Comment:
Thank you for the contribution!  Does this just spin up by default?

Reply:
yes all json files are auto populating grafana